### ISSUE TITLE
Allow passing a list of tests to run

### ISSUE THREAD

**Issue by jamesbluecrow** [mobile-dev-inc/Maestro#2130]
### Use case

I'm implementing a script to retry some tests that are flaky and I would like to be able to retry only the tests that have failed.

Ideally I would like to still use shards and provide a list of test files paths for the tests that I want to run as part of the suite.

### Proposal

I think it could be something like blow:

```
maestro test --format=junit --output=$REPORT_PATH --shard-split=4 --tests=test_1.yaml, test_2.yaml, test_3.yaml...
```

**Comment by ricardo7martins** [comment]
I believe this was introduced in 1.39.0 with #1995

**Comment by jamesbluecrow** [comment]
Right, it looks like what is expected is a list of files separated by a space.

I've gone ahead and created a PR to update the docs with this option -> https://github.com/mobile-dev-inc/maestro-docs/pull/99

**Comment by Fishbowler** [comment]
Docs merged.

I do declare this: done

🎉
